docs/modules/ROOT/pages/explanations/slos.adoc

```diff
@@ -9,6 +9,57 @@ We use the SLOs and https://sre.google/workbook/alerting-on-slos/#6-multiwindow-
 IMPORTANT: These are internal sevice level *objectives*, not service level *agreements*.
 We don't guarantee to meet these objectives at all times.
 
+== Ingress
+
+Working cluster ingress is a core requirement for a Kubernetes cluster.
+If the workloads running on the cluster aren't accessible, it might as well be down from a user perspective.
+
+=== Canary
+
+****
+*99.75% of all HTTP probes to a canary application succeed*
```

Comment:
I made an educated guess which SLO we could provide, based on https://hub.syn.tools/openshift4-slos/runbooks/ingress.html#_tune